front/components/app/blocks/Search.tsx

```diff
@@ -135,14 +135,14 @@ export default function Search({
               <Menu.Button
                 className={classNames(
                   "inline-flex items-center rounded-md py-1 text-sm font-bold",
-                  !!currentProvider?.providerId ? "px-0" : "border px-3",
+                  currentProvider?.providerId ? "px-0" : "border px-3",
```

Comment:
`!!` was removed by ESLint. It says `redundant double negation` if I put back the `!!`